File: hit_count.py
def counter_effect(hit_count):
    arr = list(hit_count)
    new_arr = []
    for x in range(len(arr)):
        new_arr.append(arr_maker(int(arr[x])))
    return new_arr

def arr_maker(num):
    arr = []
    number = 0
    while number <= num:
        arr.append(number)
        number += 1
    return arr

File: hitCount.js
function counterEffect(hitCount) {
    let newArr = [];
    let arr = hitCount.split('');
     for (let x of arr){
       newArr.push(arrayMaker(Number(x)));
     }
     return newArr;
   }
   function arrayMaker(num){
     let arr = [];
     if (num == 0){
       arr.push(0);
     }else{
       let nummy = 0;
       while(nummy <= num){
         arr.push(nummy);
         nummy++;
       }
     }
     return arr;
   }